buildlib/tools/common.sh

```diff
@@ -11,7 +11,7 @@ MVN_MODULE="dev/mvn"
 XPMEM_MODULE="dev/xpmem-90a95a4"
 PGI_MODULE="hpc-sdk/nvhpc/21.2"
 GCC_MODULE="dev/gcc-10.1.0"
-ARM_MODULE="arm-compiler/armcc-19.0"
+ARM_MODULE="arm-compiler/armcc-22.1_RHEL7"
```

Comment:
what if we run on non-RHEL7?

Reply:
It works on Ubuntu too.

Reply:
can we remove RHEL7 from the name?

Reply:
Merged (internal link) 
https://gitlab-master.nvidia.com/nbu-swx/hpc/swx-infrastructure/-/merge_requests/4
And renamed here.